lib/mcb.rb

```diff
@@ -354,7 +354,7 @@ def process_opt_changed_since(opts, url)
           CGI.unescape(opts[:'changed-since']),
           '%FT%T.%NZ'
         ) rescue nil
-        changed_since ||= DateTime.zone.parse(opts[:'changed-since']).utc
+        changed_since ||= DateTime.parse(opts[:'changed-since']) # rubocop:disable Rails/TimeZone
```

Comment:
don't you need the enable bit afterwards?

Reply:
[The docs](https://docs.rubocop.org/en/stable/configuration/#disabling-cops-within-source-code) seem to imply that for single line disabling, it's not necessary to re-enable explicitly.